packages/lib/convertToNewDurationType.ts

```diff
@@ -0,0 +1,43 @@
+export const MINUTES_IN_HOUR = 60;
+export const MINUTES_IN_DAY = 1440;
+export const HOURS_IN_DAY = 24;
+
+export default function convertToNewDurationType(
+  prevType: string | "minutes" | "hours" | "days",
+  newType: string | "minutes" | "hours" | "days",
```

Comment:
Agreed with @hariombalhara. Adding `string` defeats the purpose of setting an intersection.

```suggestion
  prevType: "minutes" | "hours" | "days",
  newType: "minutes" | "hours" | "days",
```